README.rst

```diff
@@ -33,44 +46,74 @@ Features
 
 * Combine Microsoft Azure quantum simulators with PennyLane's automatic differentiation and optimization.
 
+.. installation-start-inclusion-marker-do-not-remove
 
 Installation
 ============
 
-PennyLane-qsharp requires both PennyLane and the Microsoft Quantum Development Kit. To install the
-Microsoft QDK and IQ#, `see the intruction details <https://docs.microsoft.com/en-us/quantum/install-guide/index?view=qsharp-preview>`_
-provided by Microsoft.
+This plugin requires Python version 3.6 and above, as well as PennyLane
+and Q#. Installation of this plugin, as well as all dependencies, can be done using ``pip``:
```

Comment:
Is this line actually needed at all, since the requirements are already written clearly under Dependencies below?

```suggestion
Installation of this plugin, as well as all dependencies, can be done using ``pip``:
```

Reply:
True! Will delete.